mcs/class/referencesource/mscorlib/system/type.cs

```diff
@@ -1249,7 +1249,6 @@ public virtual Type[] GetGenericParameterConstraints()
             get {return IsCOMObjectImpl();}
         }
 
-#if FEATURE_COMINTEROP || MONO_COM
```

Comment:
"if (type.IsWindowsRuntimeObject && type.IsImport)"

IsWindowsRuntimeObject is defined behind a define removed.